firstvoices/backend/models/import_jobs.py

```diff
@@ -101,7 +100,7 @@ class Meta:
         get_user_model(), blank=True, null=True, on_delete=models.PROTECT
     )
 
-    data = models.ForeignKey(File, null=True, on_delete=models.SET_NULL)
+    data = models.ForeignKey("backend.File", null=True, on_delete=models.SET_NULL)
```

Comment:
Added string to refer to model instead of object to prevent circular import issue.

Reply:
Idea: what if we used a string reference for all the import_job field definitions instead? Would that make anything easier? Maybe avoid the need to extract files.py and edit the migrations?